app/guid-node/metadata/-components/metadata-select/template.hbs

```diff
@@ -1,16 +1,16 @@
 <div local-class='metadata-select-container {{if this.isMobile 'mobile'}}'>
     <div local-class='title' data-test-select-title>
-        {{ this.template.schemaName }}
+        {{ @template.schemaName }}
     </div>
     <div local-class='description' data-test-select-description>
-        {{ this.template.template.description}}
+        {{ @template.template.description}}
     </div>
     <div local-class='action'>
         <Button
             data-test-select-button
             aria-label= {{ t 'metadata.add-flow-select' }}
```

Comment:
Would remove this aria-label as well, as this this button already has visible text. And the aria-label uses `add-flow-select` while the button's visible text uses `add-flow.select`, so the aria-label would be broken here anyways